bb/logmsg.c

```diff
@@ -48,6 +48,8 @@ static int level_to_syslog(loglvl lvl) {
             return LOG_DEBUG;
         case LOGMSG_INFO:
             return LOG_INFO;
+        case LOGMSG_USER:
+            return LOG_USER;
```

Comment:
Sorry, I don't understand what that means. In logmsg.c we do the following: 
107         int syslog_level = level_to_syslog(lvl);
108         syslog(syslog_level, "%s", msg);
If lvl is LOGMSG_USER it will currently return critical, and that does not look right to me.